token/services/network/fabric/processor.go

```diff
@@ -65,7 +62,7 @@ func (r *RWSetProcessor) Process(req fabric.Request, tx fabric.ProcessTransactio
 	case "init":
```

Comment:
Maybe we can add a comment to explain why init stays (because we don't know the txid of the chain code-upgrade tx's) and why default returns nil (because it's processed by the filter)